docs/kusion/4-configuration-walkthrough/8-monitoring.md

```diff
@@ -26,93 +26,62 @@ import catalog.models.schema.v1.workload as wl
 import catalog.models.schema.v1.monitoring as m
 ```
 
-## Project-level configurations
+## Workspace configurations
```

Comment:
Do not include workspace configuration, only contain configuration for app developer

Reply:
done